doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -69,6 +69,13 @@ nRF9160: Asset Tracker v2
 
 * |no_changes_yet_note|
 
+nRF Desktop
+-----------
+
+* Added:
+
+  * Added bootloader variant as a DFU module variant for config channel communication. The new implementation is not backwards compatible.
```

Comment:
> The new implementation is not backwards compatible.

Why? You don't need to check the variant. Everything else will be working as it was and old script should work without any issues.